backend/store/etcd/event_store_test.go

```diff
@@ -50,7 +50,7 @@ func TestEventStorage(t *testing.T) {
 
 		// Get all events from an unexisting org
 		ctx = context.WithValue(ctx, types.OrganizationKey, "acme")
-		ctx = context.WithValue(ctx, types.EnvironmentKey, "*")
+		ctx = context.WithValue(ctx, types.EnvironmentKey, types.EnvironmentKeyAll)
```

Comment:
Looks like `types.EnvironmentKeyAll` does not exist, I assume it's a typo and should be `types.EnvironmentTypeAll`?

Reply:
easy enough, and kinda not surprised I missed this in the course of typing EnvironmentTypeAll like 100 times.